invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Document/Forms/DocumentEditor/components/DocumentForm/documentSubmitSerializer.js

```diff
@@ -1,6 +1,16 @@
 export default values => {
   const submittingValues = { ...values };
-  submittingValues['tag_pids'] = submittingValues['tags'].map(tag => tag.pid);
-  delete submittingValues['tags'];
+
+  if (submittingValues.tags) {
+    submittingValues.tag_pids = submittingValues.tags.map(tag => tag.pid);
+  }
+
+  delete submittingValues.tags;
```

Comment:
we have to ignore these keys and refs for sure in the loaders.

Reply:
This is already taken care of in the loader.